client/build.gradle.kts

```diff
@@ -60,6 +61,13 @@ dependencies {
     testProtobuf("io.spine.tools:spine-tool-base:$spineBaseVersion")
 }
 
+dart {
+    tasks {
+        registerBuildTasks()
```

Comment:
Can we turn it into this?
```
register(build(), publish())
```
It's two "register" (and I want to have just one), and two redundant "Tasks" (which is the context in which we operate already) to kill.